src/Constants.js

```diff
@@ -0,0 +1,22 @@
+export const APP_TODO_TITLE = "TODOS";
+export const STORAGE_KEY = "TodoList";
+export const PLACE_HOLDER = "할 일을 입력하세요!";
+
+export const ALERT_MESSAGE_EDIT = "현재 Todo를 수정 중입니다. 🖐️ 수정을 완료 후 진행해 주세요!";
+export const ALERT_MESSAGE_INPUT = "2글자 이상 입력해주세요. ✏️";
+
+export const FILTER_OPTIONS = [
+  { option: "All", text: "전체 항목", isDefault: true },
+  { option: "Todo", text: "해야할 일", isDefault: false },
+  { option: "Completed", text: "완료한 일", isDefault: false },
+];
+
+export const ALERT_MESSAGE_SET_LIST = "새로운 TodoList 업로드에 실패했습니다. 🥹";
+export const ALERT_MESSAGE_UPDATE_LIST = "List 를 업데이트하는 과정에서 오류가 발생했습니다! 😡";
+export const ALERT_MESSAGE_GET_LIST = "Todo 들을 가져오다가 실패했습니다. 🥹";
```

Comment:
아래와 같이 객체로 한번 묶어서 사용해보셔도 좋을거 같아요

```suggestion
export const ALERT_MESSAGE = {
  SET_LIST: ...,
  UPDATE_LIST: ...
}
```

Reply:
앟.. 이런 방법이.. 별도로 분리할 생각만 했는데 감사합니다